### PR TITLE
jquery xss vulnerability issue fix

### PR DESCRIPTION
**Pull request by fiyazbinhasan**
fix of issue [8482](https://github.com/aspnet/Docs/issues/8482)
both 2.0 and 2.1 samples have been modified. please review/test before merging. 

N.B: to be able to run the app, first you need to add the following two lines before `UseMvc()` in the startup. (since the same sample has been used to demostrate an interface less api and later used with a jquery ui).

`app.UseDefaultFiles();`
` app.UseStaticFiles();`

test case: try adding a self-running script in place of a name of a specific todo `<script>window.alert("I am vulnerable")</script>`

### Previously 

previously

### After fix

now

[Internal Review Page](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-2.1&branch=pr-en-us-9415#call-the-web-api-with-jquery)

### CODE REVIEW

**Comment by fiyazbinhasan** [comment]
@Rick-Anderson fix of [8482](https://github.com/aspnet/Docs/issues/8482). please check :)

**Comment by fiyazbinhasan** [comment]
vs code auto-formatted the js

**Comment by fiyazbinhasan** [comment]
@scottaddie please check

File: aspnetcore/tutorials/first-web-api/samples/2.1/TodoApi/wwwroot/site.js [hunk 1]
```diff
@@ -1,109 +1,144 @@
...
-            $('#todos').empty();
-            getCount(data.length);
-            $.each(data, function (key, item) {
-                const checked = item.isComplete ? 'checked' : '';
+  $.ajax({
+    type: "GET",
+    url: uri,
+    cache: false,
+    success: function(data) {
+      var tBody = $("#todos");
```
Comment (scottaddie) [inline aspnetcore/tutorials/first-web-api/samples/2.1/TodoApi/wwwroot/site.js]: var --> const

File: aspnetcore/tutorials/first-web-api/samples/2.1/TodoApi/wwwroot/site.js [hunk 2]
```diff
@@ -1,109 +1,144 @@
...
-            });
+      $(tBody).empty();
 
-            todos = data;
-        }
-    });
+      getCount(data.length);
+
+      $.each(data, function(key, item) {
+        var tr = $("<tr></tr>")
```
Comment (scottaddie) [inline aspnetcore/tutorials/first-web-api/samples/2.1/TodoApi/wwwroot/site.js]: var --> const

File: aspnetcore/tutorials/first-web-api/samples/2.0/TodoApi/wwwroot/site.js [hunk 3]
```diff
@@ -1,119 +1,144 @@
...
-            });
+      $(tBody).empty();
 
-            todos = data;
-        }
-    });
+      getCount(data.length);
+
+      $.each(data, function(key, item) {
+        var tr = $("<tr></tr>")
```
Comment (scottaddie) [inline aspnetcore/tutorials/first-web-api/samples/2.0/TodoApi/wwwroot/site.js]: var --> const

File: aspnetcore/tutorials/first-web-api/samples/2.0/TodoApi/wwwroot/site.js [hunk 4]
```diff
@@ -1,119 +1,144 @@
...
-            $('#todos').empty();
-            getCount(data.length);
-            $.each(data, function (key, item) {
-                const checked = item.isComplete ? 'checked' : '';
+  $.ajax({
+    type: "GET",
+    url: uri,
+    cache: false,
+    success: function(data) {
+      var tBody = $("#todos");
```
Comment (scottaddie) [inline aspnetcore/tutorials/first-web-api/samples/2.0/TodoApi/wwwroot/site.js]: var --> const

**Comment by scottaddie** [comment]
@fiyazbinhasan The snippet changes look great. One more round of updates, and then this looks good.

**Comment by scottaddie** [comment]
@fiyazbinhasan Thank you for your help!

**Comment by fiyazbinhasan** [comment]
Anytime :)

**Pull request merged**